code/datums/loadout/item_representation/_item_representation.dm

```diff
@@ -63,22 +64,59 @@
 	if(istype(item_to_copy, /obj/item/storage/internal))
 		bypass_vendor_check = TRUE
 	var/item_representation_type
+	var/list/item_bypassable = list()
+	var/bypassable = FALSE
+	for(var/type in item_to_copy.starting_items)
+		item_bypassable[type] = item_to_copy.starting_items[type] > 0 ? item_to_copy.starting_items[type] : 1
 	for(var/atom/thing_in_content AS in item_to_copy.contents)
 		if(!isitem(thing_in_content))
 			continue
-		if(!is_savable_in_loadout(thing_in_content))
+		bypassable = FALSE
+		if(item_bypassable[thing_in_content.type] > 0)
+			item_bypassable[thing_in_content.type]--
+			bypassable = TRUE
+		if(!bypassable && !is_savable_in_loadout(thing_in_content, loadout, item_to_copy))
```

Comment:
and this

Reply:
We have a list of item that are free for this storage, aka they are already filling up the storage when you buy them.
If the medbelt holds 3 brute pack when instantiated, we allow for 3 brute pack to be created for three. Once you go above 3, the health pack will no longer bypass vendor check and the loadout seller will look in all vendors like usual